clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/ValidateSchemaAndBuildDictOutputFormat.java

```diff
@@ -30,15 +30,12 @@
 public class ValidateSchemaAndBuildDictOutputFormat extends AvroOutputFormat {
```

Comment:
[Comment in random place to start a conversation]

Can you update the PR description? I don't think the issue has anything to do with the sticky bit any more and the permission checks are the same as POSIX in this case now

Reply:
Done.